src/LodeRunner.Core/Models/TestRunPayload.cs

```diff
@@ -50,7 +42,7 @@ public class TestRunPayload : BasePayload
         /// <value>
         /// The CreatedTime.
         /// </value>
-        public DateTime CreatedTime { get => this.testRun.CreatedTime; set => this.SetField(this.testRun, value); }
+        public DateTime? CreatedTime { get; set; }

```

Comment:
I noticed that CreatedTime and StartTime have the required attribute in the TestRun model class.  Does a default value for these properties get set in when the TestRun object is created from the payload?

Reply:
Good question, we do have Automapper for that and on this PR I also refactored the Extension method "IgnoreNullFields" to check for Field.Value not null instead PropertiesChangedList to do the mapping when creating TestRuns from TestRunPayload